### ISSUE TITLE
The performance is worse than the reported metrics on tiny imagenet

### ISSUE THREAD

**Issue by lihuiknight** [POSTECH-CVLab/PyTorch-StudioGAN#61]
Dear author,

I run ContraGAN with batch=256 on 8xV100, the best IS is 12.49601936340332 and the best FID is 30.344396454496405 while the reported IS and FID in your github is 13.494 and 27.027.   Are the perturbation on performance caused by the model's variance or by the hyperparameters? What's your suggestion?

**Comment by MINGUKKANG** [comment]
Hi,

I think that the discrepancy is from the number of GPUs. 

I used 4 GPUs to train ContraGAN on Tiny ImageNet without sync_bn. 

My deep learning machine processes 256 (1024/4GPUs) images per GPU. Thus, the 256 images are used to calculate empirical means and variances for normalizating features in each device.

Also, I attach the results from three runs of ContraGAN on Tiny ImageNet here.

28.16631825 +- 0.8767019505

Thank you.

Best,

**Comment by lihuiknight** [comment]
ok, i see. if so, i need to specify the batch size as 1024, so that the expected mean and variance will be exactly the same as yours.
